modules/search/instant-search/lib/query-string.js

```diff
@@ -27,6 +27,51 @@ export function setSearchQuery( searchValue ) {
 	pushQueryString( encode( query ) );
 }
 
+export function getSearchSort() {
+	const query = getQuery();
+	const order = 'order' in query ? query.order : 'DESC';
+	const orderby = 'orderby' in query ? query.orderby : 'relevance';
+	let sort;
+	switch ( orderby ) {
+		case 'date':
+			if ( order === 'ASC' ) {
+				sort = 'date_asc';
+			} else {
+				sort = 'date_desc';
+			}
+			break;
+		case 'price':
+			if ( order === 'ASC' ) {
+				sort = 'price_asc';
+			} else {
+				sort = 'price_desc';
+			}
+			break;
+		case 'rating':
+			if ( order === 'ASC' ) {
+				sort = 'rating_asc';
+			} else {
+				sort = 'rating_desc';
+			}
+			break;
+		case 'recency':
```

Comment:
Do we not want to have ascending/descending options for results ordered by recency, keyword, popularity, and relevance?

Reply:
We can really only do descending when you sort by any calculated score. Search engines are built to be fast to compute the top results only.